plugins/woocommerce/src/StoreApi/Schemas/V1/CheckoutSchema.php

```diff
@@ -268,6 +268,10 @@ protected function get_additional_fields_response( \WC_Order $order ) {
 
 		$additional_field_schema = $this->get_additional_fields_schema();
 		foreach ( $fields as $key => $value ) {
+			if ( ! isset( $additional_field_schema[ $key ] ) ) {
+				unset( $fields[ $key ] );
+				continue;
+			}
```

Comment:
This shouldn't happen because we only load registered fields and all registered fields get a schema, but I added it regardless.

Reply:
Thanks for adding that, I agree it shouldn't happen but if it does it's better to have the check in place.